### PR TITLE
Circle radius and center coordinates 

### PR DESCRIPTION
**Pull request by NEEHITGOYAL**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #("mention issue no.")
#307 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [x] Bug fix (non-breaking change which fixes an issue)
* [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [x] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Python/circle_detection/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+A Python Script to detect shapes,shapes area, color,centroid from images.
+
+## How to execute this Script
+
++ Run `pip install opencv-python` to install the required packages.
++ Add the image to same folder with the name **img.jpeg**
++ Run the script using `python3 circle.py`
++ The script will printout circle radius centre coordinates as well as image with same
+
+
```
Comment (yashaswibiyahut) [inline Python/circle_detection/README.md:18]: Add example or use case in readme file

File: Python/circle_detection/circle.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+gray = cv.medianBlur(gray, 5)
+#hough circle detection method for circle detection
+circles = cv.HoughCircles(gray, cv.HOUGH_GRADIENT, 1, 20,
+                          param1=50, param2=30, minRadius=0, maxRadius=0)
+detected_circles = np.uint16(np.around(circles))
+for (x, y ,r) in detected_circles[0, :]:
+    cv.circle(output, (x, y), r, (0, 0, 0), 3)
+    cv.circle(output, (x, y), 2, (0, 255, 255), 3)
+    print("x={},y={},radius = {}\n".format(x,y,r))
+
```
Comment (yashaswibiyahut) [inline Python/circle_detection/circle.py]: add proper comments

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

**Comment by yashaswibiyahut** [comment]
Squash the commits please

**Comment by NEEHITGOYAL** [comment]
i have tried to rebase but i don't know what is happening actually i am squashing first time so i  searched 
and done it but here it is not showing anything

**Comment by tejan-singh** [comment]
@NEEHITGOYAL if you are finding it difficult, then close this pull request and create a new one

**Comment by NEEHITGOYAL** [comment]
i am opening new pr for same

**Pull request closed without merging**